src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__box calendar__box--month-length--29 calendar__box--start-day--sun">
+        <div class="calendar__day calendar__day--1"></div>
```

Comment:
don't create separate class for each calendar day
use nth-child instead of using exact number for each day in class name